### PR TITLE
libsecureboot: be more verbose about validation failures

### PR DESCRIPTION
**Pull request by stephane-rochoy-stormshield**
Use `find_error_name` to be more explicit about the reason of validation failures. E.g., 

    cannot verify: /boot/pass_manifest.rcerts: Validation failed, Chain could not be linked to a trust anchor. (BR_ERR_X509_NOT_TRUSTED)

### CODE REVIEW

**Comment by bsdimp** [comment]
I think these changes are good.

**Comment by sgerraty** [comment]
Looks ok to me

File: lib/libsecureboot/vets.c [hunk 1]
```diff
@@ -568,9 +568,15 @@ verify_signer_xcs(br_x509_certificate *xcs,
 			ve_error_set("Validation failed, certificate not valid as of %s",
 			    gdate(date, sizeof(date), ve_utc));
 			break;
-		default:
-			ve_error_set("Validation failed, err = %d", err);
-			break;
+		default: {
```
Comment (igalic) [inline lib/libsecureboot/vets.c:571]: the formatting in this file looks off
did you run this through tools/build/checkstyle9.pl

**Review by igalic — CHANGES_REQUESTED** [review]

File: lib/libsecureboot/vets.c (hunk 1 above)
Comment (stephane-rochoy-stormshield) [reply lib/libsecureboot/vets.c:571]: No I don't. I'll play with it ASAP.

**Comment by emaste** [comment]
4b9d605768acabc460aa6dcfe8a1f8db35b16794

**Pull request closed without merging**